.github/workflows/metrics.yml

```diff
@@ -0,0 +1,70 @@
+on:
+  push:
+    branches: [main]
+  pull_request:
+    paths:
+      - .github/workflows/metrics.yml
+      - dart/**
+      - flutter/**
+      - metrics/**
+
+jobs:
+  metrics:
+    runs-on: macos-latest
+    strategy:
+      # We want that the matrix keeps running, default is to cancel them if it fails.
+      fail-fast: false
+      matrix:
+        platform: ['ios', 'android']
+        # Let's stick to an explicit version and update manually because a version change may affect results.
+        # If it updated automatically, it could confuse people to think the change is actually caused by the PR.
+        flutter: ['3.3.3']
```

Comment:
Should we add to `update-deps.yml` at least?

Reply:
done